### ISSUE TITLE
How can I use my own tokenizer or use tokenized results as input?

### ISSUE THREAD

**Issue by yyHaker** [hanxiao/bert-as-service#124]
**Prerequisites**

> Please fill in by replacing `[ ]` with `[x]`.

* [ ] Are you running the latest `bert-as-service`?
* [ ] Did you follow [the installation](https://github.com/hanxiao/bert-as-service#install) and [the usage](https://github.com/hanxiao/bert-as-service#usage) instructions in `README.md`?
* [ ] Did you check the [FAQ list in `README.md`](https://github.com/hanxiao/bert-as-service#faq)?
* [ ] Did you perform [a cursory search on existing issues](https://github.com/hanxiao/bert-as-service/issues)?

**System information**

> Some of this information can be collected via [this script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh).

- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
- TensorFlow installed from (source or binary):
- TensorFlow version:
- Python version:
- `bert-as-service` version: 
- GPU model and memory:
- CPU model and memory:

---

### Description

> Please replace `YOUR_SERVER_ARGS` and `YOUR_CLIENT_ARGS` accordingly. You can also write your own description for reproducing the issue.

I'm using this command to start the server:

```bash
bert-serving-start YOUR_SERVER_ARGS
```

and calling the server via:
```python
bc = BertClient(YOUR_CLIENT_ARGS)
bc.encode()
```

Then this issue shows up:

...

**Comment by hanxiao** [comment]
require some refactoring, but I see the needs. 

will do, mark it as enhancement.

**Comment by hanxiao** [comment]
But please keep in mind that the pretrained BERT Chinese is a character-based model, so if you use your own segmentation without changing the underlying BERT model, you will get a bunch of OOV.

It only makes sense when the served BERT model is *not* character-based and follows the same vocabulary as your segmentation.

**Comment by hanxiao** [comment]
fyi, this issue has been solved and the feature is available since 1.5.2, please do 
```bash
pip install -U bert-serving-server bert-serving-client
```
Usage and more info can be found in the tutorial: https://github.com/hanxiao/bert-as-service#using-your-own-tokenizer
